src/project-tests/technical-docs-tests.js

```diff
@@ -114,7 +114,23 @@ export default function createTechnicalDocsPageTests() {
                         '<a> elements ');
                 });
 
-            it('11. Each element with the class of "nav-link" should contain text that corresponds to the <header> text ' +
+            it('11. The <header> element in the navbar must come before any link (<a>) elements in the navbar.',
+                function() {
+                  const navLinks = document.querySelectorAll('#navbar a.nav-link');
+                  const header = document.querySelectorAll('#navbar header').item(0);
+                  let allAbove = true;
+
+                  navLinks.forEach(navLink => {
+                    if ( ! (header.compareDocumentPosition(navLink) & Node.DOCUMENT_POSITION_FOLLOWING)) {
```

Comment:
I would keep the `!`, right up against the value you are trying to negate, e.g. `!true` - more commonly written this way. 

Nice use of the bitwise `&` operator! I've very rarely seen a good use case for it. Looks like you read the docs carefully for this. Nice job.